src/altimate.ts

```diff
@@ -647,8 +647,12 @@ export class AltimateRequest {
         status: response.status,
         textResponse,
       });
+      let jsonResponse: any;
+      try {
+        jsonResponse = JSON.parse(textResponse);
+      } catch {}
       throw new APIError(
-        `Could not process request, server responded with ${response.status}: ${textResponse}`,
+        `Could not process request, server responded with ${response.status}: ${jsonResponse?.detail || textResponse}`,
```

Comment:
What is this solving or providing opposed to the earlier code?

Reply:
In some cases, the message shown in error notification contains json. Now I am just passing relevant details from backend error message.

![image](https://github.com/user-attachments/assets/c8c7d413-dc5f-4e42-b071-1396a39fd84c)

Reply:
What if detail doesn't exist but the repsonse is json?

Reply:
this shouldn't happen as fastapi gives error response in this format only.
@suryaiyer95 correct me if i am wrong.